### DISCUSSION TITLE
Will flashing Terraos to local storage not allow me to fix it when the school requires us to turn it in at the end of eighth grade?

### DISCUSSION THREAD

**Discussion by FunnyCheese4Me** [3kh0/ext-remover#1106] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school requires us to turn in our Chromebooks at the end of eighth grade. If I flashed terraos to the local storage, would that not allow me to fix it for the next user of my Chromebook?

**Comment by S-PScripts** [comment]
your chromebook gives the old ones to new students? that's dodgy

**Reply by FunnyCorgi** [reply]
so don't do it then?

**Comment by nathblade16** [comment, marked as answer]
you would be able to fix it by recovering chromeOS with a usb(kinda downgrading but not downgrading) and that will restore your chormebooks partition table back to stock

**Reply by FunnyCorgi** [reply]
ok thanks for letting me know.
